l/lodash/lodash_4.17.21_UBI_8.5.sh

```diff
@@ -0,0 +1,60 @@
+# -----------------------------------------------------------------------------
```

Comment:
Missed Shabang statement (#!/bin/bash -e). Causing travis check failure.

Reply:
Thanks @kandarpamalipeddi .Updating it.